sort.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+# !source env/bin/activate
+
+import sys
+import argparse
+
+parser.add_argument('infile', metavar = 'FILE', nargs = "?", help = 'input file', default='-')
+
+args = parser.parse_args()
+
+file = args.infile
+if file == "-":
+    file = sys.stdin
+    lines = []
+    for line in file:
+        lines.append(line)
+
+    ans = "\n".join(sorted(lines))
```

Comment:
Перенос строки тут лишний, из-за него в аутпуте появляются лишние переносы
![image](https://user-images.githubusercontent.com/48160953/145906018-52b8f835-7b47-4610-95a9-6217a83daeab.png)
```suggestion
    ans = "".join(sorted(lines))
```